test/irb/helper.rb

```diff
@@ -49,6 +49,19 @@ def ruby_core?
       !Pathname(__dir__).join("../../", "irb.gemspec").exist?
     end
 
+    def setup_envs(home:)
+      @backup_home = ENV["HOME"]
+      ENV["HOME"] = home
+      @backup_xdg_config_home = ENV.delete("XDG_CONFIG_HOME")
+      @backup_irbrc = ENV.delete("IRBRC")
+    end
+
+    def teardown_envs
+      ENV["HOME"] = @backup_home
+      ENV["XDG_CONFIG_HOME"] = @backup_xdg_config_home
+      ENV["IRBRC"] = @backup_irbrc_backup
```

Comment:
I think this is a typo:

```suggestion
      ENV["IRBRC"] = @backup_irbrc
```

Reply:
Thank you, fixed